inspirehep/modules/theme/views.py

```diff
@@ -658,6 +659,19 @@ def linkedaccounts():
 
 
 #
+# Record preview
+#
+
+@blueprint.route('/record/preview', methods=['POST'])
```

Comment:
Is it a problem to make it more RESTy? Like: `/record/<int:recid/preview`. Also, the HTTP method should semantically be `GET`.